bin/git/create-repository.js

```diff
@@ -0,0 +1,63 @@
+// global "date"
```

Comment:
This is a file for testing, should it be in the `__tests__` directory?

Reply:
sure